src/Valuestore.php

```diff
@@ -48,12 +47,12 @@ protected function setFileName(string $fileName)
     /**
      * Put a value in the store.
      *
-     * @param string|array    $name
-     * @param string|int|null $value
+     * @param array|string                $name
+     * @param array|string|float|int|null $value
```

Comment:
Can't we just typehint mixed? Would look nice, and we don't need the doc block anymore.

Reply:
I kind of agree, however, the `mixed` type is wider than the union type we need here (It includes objects and resources. And objects and resources cannot be proceed by the package). So, I see to options here:
- leave it as it is with a type-hint in the docblock
- add this union type as a type-hint to the $value parameter, but it's too wordy though (however I would prefer this approach because it validates input and leaves no chance for misuse)

What do you think?

Reply:
UP: I've uploaded my take on how it could look like with long wordy union types. But I would prefer this approach. It's explicit and it follows the last trends (to use native types and remove superfluous PHP annotations).